lib/ClusterProvider.py

```diff
@@ -0,0 +1,37 @@
+import common
+import subprocess
+import os
+from robot.api import logger
+
+ROBOT_LIBRARY_SCOPE = 'SUITE'
+AUTH_COMMAND = ''
+
+
+def auth_wrap(cmd):
+    return AUTH_COMMAND+' && '+cmd
+
+
+class ClusterProvider(common.CommandRunner):
```

Comment:
We should keep the same functions names as in Kind.py - this is going to reduce the number of changes we have in robot files.

Reply:
Done